renderdoc/os/posix/posix_process.cpp

```diff
@@ -41,14 +41,16 @@ int GetIdentPort(pid_t childPid);
 
 static const string GetAbsoluteAppPathFromName(const string &appName)
 {
+  string appPath;
+
   // If the application name contains a slash character convert it to an absolute path and return it
   if(appName.find("/") != string::npos)
   {
     char realpathBuffer[PATH_MAX];
     string appDir = dirname(appName);
     string appBasename = basename(appName);
     realpath(appDir.c_str(), realpathBuffer);
-    string appPath(realpathBuffer);
+    appPath.assign(realpathBuffer, strlen(realpathBuffer));
```

Comment:
It didn't occur to me that you could assign a char* to a string like that.  Sure enough, it is supported by operator=.  I'll update it, thanks.